# Generated for NaoQi method ALBonjour.declare
# Declares NaoQi using bonjour.

---


